primefaces/src/main/frontend/packages/datepicker/datepicker/src/jquery.ui.prime.datepicker.cjs

```diff
@@ -1625,7 +1625,7 @@ $.widget("prime.datePicker", {
                 lastDate = weekDates[6],
                 cellClass = firstDate.otherMonth && lastDate.otherMonth && !this.options.showOtherMonths ? ' ui-datepicker-other-month-hidden' : '';
 
-            weekHtml += '<td class="ui-datepicker-weeknumber' + cellClass + '"><span class="ui-state-disabled">' +
+            weekHtml += '<td class="ui-datepicker-weeknumber' + cellClass + '"><span class="ui-state-disabled" aria-disabled="true">' +
```

Comment:
Probably on the span the aria-disabled is not necessary, because it has no meaning to the screen reader